Applications/Test0/minimalapp.hpp

```diff
@@ -0,0 +1,50 @@
+#include <QApplication>
```

Comment:
Maybe we should update this class to follow the rest of the codebase and use `m_` as a prefix for members. (Remember that the standard frowns upon using underscores as the first character of an identifier...)